### PR TITLE
Get PTVS building for D17

### PR DESCRIPTION
**Pull request by rchiodo**
This makes PTVS build (and run) with D17.

I'm going to do the setup authoring work separately after I get this to go through.

### CODE REVIEW

File: Common/Tests/AssemblyInfoCommon.cs [hunk 1]
```diff
@@ -47,6 +47,9 @@ class AssemblyVersionInfo {
 #elif DEV16
     public const string VSMajorVersion = "16";
     public const string VSVersionSuffix = "2019";
+#elif DEV16
```
Comment (bschnurr) [inline Common/Tests/AssemblyInfoCommon.cs]: DEV17 ?

Comment (bschnurr) [reply Common/Tests/AssemblyInfoCommon.cs]: nevermind i think you fixed this

**Comment by rchiodo** [comment]
@bschnurr did you have any more comments? I think I'm ready to submit now. At least it builds for D17. I'll work on getting this into VS proper next.

File: Python/Product/PythonTools/PythonTools.csproj [hunk 2]
```diff
@@ -183,8 +165,8 @@
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Workspace" />
     <Reference Include="Microsoft.VisualStudio.Workspace.VSIntegration.Contracts" />
-    <Reference Include="Newtonsoft.Json, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
-      <HintPath>$(PackagesPath)\Microsoft.VSSDK.BuildTools\tools\vssdk\bin\lib\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (bschnurr) [inline Python/Product/PythonTools/PythonTools.csproj:169]: did this fix the Newtonsoft.Json warning?

File: Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj [hunk 3]
```diff
@@ -73,7 +75,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Common\Common.csproj">
       <Project>{b3db0521-d9e3-4f48-9e2e-e5ecae886049}</Project>
-      <Name>Microsoft.PythonTools.Common</Name>
+      <Name>Common</Name>
```
Comment (bschnurr) [inline Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj:78]: not sure why you shortened the name. previous one is a bit more clear

**Review by bschnurr — APPROVED** [review]

File: Python/Product/PythonTools/PythonTools.csproj (hunk 2 above)
Comment (rchiodo) [reply Python/Product/PythonTools/PythonTools.csproj:169]: I wish. This was necessary to get it to build with D16

File: Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj (hunk 3 above)
Comment (rchiodo) [reply Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj:78]: This was necessary to get this to find it when building dependencies. The project reference had to match the project name.

**Pull request merged**